src/components/Inputs/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react'
+import * as S from './styles'
+import Eye from '@/assets/img/eye.png'
+import CloseEye from '@/assets/img/close-eye(TEMPORARY).png'
```

Comment:
Sugestão: Acho que é uma boa pular uma linha pra diferenciar os níveis dos imports. Mais ou menos assim:
```suggestion
import { useState } from 'react'

import * as S from './styles'

import Eye from '@/assets/img/eye.png'
import CloseEye from '@/assets/img/close-eye(TEMPORARY).png'
```

Reply:
Nice, feito em https://github.com/Pews-DEV/book-writer-frontend/pull/16/commits/57ab5bd50a2d57d6ba8465e9036af689ad6b7cb8